tasks/SODM/SODM_pickTrials.m

```diff
@@ -0,0 +1,22 @@
+function [trial, config] = SODM_pickTrials(data) 
+
+% from pickAndEvaluateTrial 
+
+% want to pick a trial from each of the 4 types of blocks 
+% use getOneBlockPerCondition 
+getOneBlockPerCondition(data.blocks) 
```

Comment:
Either delete this line (as it merely prints one possible selection to stdout) or save it into a variable and use it in the for loop.